package.json

```
@@ -4,12 +4,13 @@
   "description": "Template for generating Go module based on async api",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"no test specified yet\"",
+    "test": "npm run testComponents",
     "release": "semantic-release",
     "lint": "eslint --max-warnings 0 --fix --config .eslintrc .",
     "generate:readme:toc": "markdown-toc -i README.md",
     "generate:assets": "npm run generate:readme:toc",
-    "bump:version": "npm --no-git-tag-version --allow-same-version version $VERSION"
+    "bump:version": "npm --no-git-tag-version --allow-same-version version $VERSION",
+    "testComponents": "rimraf test/temp && jest --coverage ./test/components"
```

```suggestion
    "test:components": "rimraf test/temp && jest --coverage ./test/components"
```

small suggestion for script name, just to be consistent and follow the standards